assets/src/js/app.js

```diff
@@ -12,6 +12,7 @@ import {removeNoPostText} from './query-no-posts';
 import {removeRelatedPostsSection} from './remove_related_section_no_posts';
 import {setupCountrySelector} from './country_selector';
 import {setupTransparentNavHomepage} from './homepage_transparent_nav';
+import {markActionsAsCompleted} from './completed_actions';
```

Comment:
What do you think to import this module from [here](https://github.com/greenpeace/planet4-master-theme/blob/39242bac9b121aa804f2756b7e45eb3b9afed8ea/assets/src/js/resistance-hub-campaign.js) ?

Reply:
Nevermind, it's not working 🤦‍♀️ gonna look into it again

Reply:
I think it's better from JS. Why? Because the `markActionsAsCompleted` has a strong dependency of the resistance hub js logic. It cannot work isolated. Wha do uou think?